src/main/java/models/ClientRequest.java

```diff
@@ -0,0 +1,56 @@
+package models;
+
+public class ClientRequest {
+    public enum requestType {
```

Comment:
I change my mind.. can these be a little more descriptive?
e.g. ADD --> ADD_RIDE_REQUEST
etc.